src/deploy/osp_deployer/director.py

```diff
@@ -3570,7 +3692,7 @@ def _get_default_compute_services(self):
 
 
 if __name__ == "__main__":
-    settings = Settings("/root/R62.ini")
+    settings = Settings("/root/r88.ini")
     director = Director()
```

Comment:
can you remove all this, should not be here.

Reply:
Done.